std_msgs/Header header
float64[7] joint_angles
float64 gripper_pos
string command_type